src/lib/connections.ts

```diff
@@ -197,3 +197,53 @@ export function signPayloadWithKeyring(signingAccount: KeyringPair, payload: any
     return 'ERROR ' + e.message;
   }
 }
+
```

Comment:
Might not be due to this PR as it is in the unchanged code, but got this odd... Success+Failure result.

<img width="954" alt="image" src="https://github.com/LibertyDSNP/provider-dashboard/assets/1252199/abdc2d1c-081d-41e4-bb5a-38f59c013dbc">

Reply:
This was brought over from signing-ui - maybe we should just get rid of the timeout. I don't think it's actually useful.

Reply:
Just hiding the timeout message for now.